website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
@brianjgeiger That's very good advice, totally open my eyes. @sloria change made in 8baac87. Thanks a lot.